boards/arm/mec2016evb_assy6797/doc/mec2016evb_assy6797.rst

```diff
@@ -0,0 +1,212 @@
+     :align: center
+     :alt: MEC2016 EVB ASSY 6797
+
+Hardware
+********
+
+- MEC1701QC2SZ ARM Cortex-M4F Processor
+- 480 KB Ram and 64 KB boot Rom
+- 2 Microchip BC-Link Interconnection bus
+- Keyboard interface
+- ADC & GPIO headers
+- UART0 and UART1
+- FAN0, FAN1, FAN2 headers
+- FAN PWM interface
+- Jtag and Trace ports
+- PECI interface 3.0
+- I2C voltage translator
+- 10 SMBUS headers
+- 3 UDP I2C headers
+- VCI interface
+- 5 independent Hardware Driven PS/2 Ports
+- eSPI header
+- LPC sideband headers
+- 4 Breathing/Blinking LEDs
+- 2 Sockets for SPI NOR chips
+- One reset and VCC_PWRDGD pushbuttons
+
+For more information about the SOC please visit:
+
+- `MEC170x Reference Manual`_
+
+Supported Features
+==================
+
+The mec2016evb_assy6797 board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by Zephyr (at the moment)
+
+The default configuration can be found in the Kconfig file:
+
+        ``boards/arm/mec2016evb_assy6797/mec2016evb_assy6797_defconfig``
+
+
+Connections and IOs
+===================
+
+Microchip to provide the schematic for this board.
+
+System Clock
+============
+
+The MEC1701 MCU is configured to use the 48Mhz internal oscilator with the
+on-chip PLL to generate a resulting EC clock rate of 12 MHz. See Processor clock
+control register (chapter 4 in user manual)
+
+Serial Port
+===========
+
+UART0 is configured for serial logs.
+
+Jumper settings
+***************
+
+Please follow the jumper settings below for in order to properly demo this
+board. Advanced users may deviate from this recomendation.
+
+Jump setting for MEC2016 EVB Assy 6797 Rev A1p0
+===============================================
+
+Power-related jumpers.
+
++-------+------+------+------+------+------+-------+-------+
+| JP31  | JP32 | JP35 | JP36 | JP49 | JP50 | JP54  | JP55  |
++=======+======+======+======+======+======+=======+=======+
+| 1-2   | 1-2  | 1-2  | 1-2  | 1-2  | 1-2  | 1-2   | 1-2   |
++-------+------+------+------+------+------+-------+-------+
+
++-------+-------+------+------+------+-------+
+| JP56  | JP57  | JP58 | JP60 | JP61 | JP102 |
++=======+=======+======+======+======+=======+
+| 1-2   | 1-2   | 1-2  | 1-2  | 1-2  | 2-3   |
++-------+-------+------+------+------+-------+
+
+This jumpers configure VCC Powergood, nRESETI and JTAG_STRAP respectively.
```

Comment:
It appears that ``power good`` should be two words:
```suggestion
These jumpers configure VCC Power good, nRESETI, and JTAG_STRAP respectively.
```

Reply:
done